javascript-local-storage/data.js

```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+
+var todos = [];
+
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+todos = JSON.parse(previousTodosJSON);
```

Comment:
What happens to the value of `todos` if there are no `previousTodosJSON`?

This may work but could possibly cause errors in a more complex application if the data type of `todos` changes to something you're not expecting.

How can we check for that and ensure that `todos` stays as an array in that case?

Reply:
I edited the code, but I'm not sure if it's correct. I might be thinking too hard about it.